LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_fractional_coordinates.html

```diff
@@ -45,8 +45,10 @@
             }
 
             function checkPointerEventCoordinates(event) {
-              if (event.clientX != Math.floor(event.clientX) ||
-                    event.clientY != Math.floor(event.clientY))
+              if ((event.clientX != Math.floor(event.clientX) ||
+                    event.clientY != Math.floor(event.clientY)) &&
+                    (event.pageX != Math.floor(event.pageX) ||
+                    event.pageY != Math.floor(event.pageY)))
```

Comment:
```suggestion
              if ((event.clientX != Math.floor(event.clientX) || event.clientY != Math.floor(event.clientY))
                    && (event.pageX != Math.floor(event.pageX) || event.pageY != Math.floor(event.pageY)))
```
I find this easier to read